WhaleSpotting/Controllers/TripPlannerController.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Models.Response;
+using WhaleSpotting.Services;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
+[Route("plan-trip")]
+public class TripPlannerController : ControllerBase
+{
+    private readonly ITripPlannerService _tripPlannerService;
+
+    public TripPlannerController(ITripPlannerService tripPlannerService)
+    {
+        _tripPlannerService = tripPlannerService;
+    }
+
+    [HttpGet("")]
+    public ActionResult<List<TripPlannerResponse>> GetNearBySightings([FromQuery] TripPlannerRequest input)
```

Comment:
Can we rename this parameter so it is self-explanatory?
```suggestion
    public ActionResult<List<TripPlannerResponse>> GetNearBySightings([FromQuery] TripPlannerRequest tripPlannerRequest)
```

Reply:
changed